conf/upgrade.yaml.template

```diff
@@ -0,0 +1,24 @@
+UPGRADE:
+  # Base version of the Satellite, Capsule.
+  # FROM_VERSION: "6.8"
+  # Target version of the Satellite, Capsule.
+  # TO_VERSION: "6.9"
+  # Satellite, Capsule hosts RHEL operating system version.
+  # OS: "rhel7"
+  # Capsule's activation key, These keys will only available if we create the satellite vm -
```

Comment:
Think you have a typo. Is it supposed to say "These keys will only "be" available?